config/kubernetes/sig-docs/teams.yaml

```
@@ -277,6 +277,7 @@ teams:
     - seokho-son # L10n: Korean
     - tnir # L10n: Japanese
     - truongnh1992 # L10n: Vietnamese
+    - xichengliudui
```

Please add a comment indicating that permissions are based on localization:
```suggestion
    - xichengliudui # L10n: Chinese
```

---

I haven't been active this week, but I've been trying